dev-libs/libnslog/libnslog-0.1.3.recipe

```diff
@@ -49,15 +49,15 @@ TEST_REQUIRES="
 
 BUILD()
 {
-	for linkage in lib-static lib-shared; do
+	for linkage in lib-shared; do
```

Comment:
The loop could have been removed, now that there is only one kind of linkage (below too).